data/os/RedHat-7.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-cis::redhat::rhel7::apply_rhel7_level_one_scored: false
-cis::redhat::rhel7::apply_rhel7_level_one_notscored: false
-cis::redhat::rhel7::apply_rhel7_level_two_scored: false
-cis::redhat::rhel7::apply_rhel7_level_two_notscored: false
\ No newline at end of file
+cis::redhat::redhat7::v1::rule_1_1_1::is_active: false
+
+cis::redhat::redhat7::v2::rule_1_1_1_1::is_active: true
```

Comment:
I thought that it will be rhel7 instead of redhat7. 
Is this because of facts?

Reply:
ya, from facter -p

```
os => {
  architecture => "x86_64",
  family => "RedHat",
  hardware => "x86_64",
  name => "RedHat",
  release => {
    full => "7.0",
    major => "7",
    minor => "0"
  },
  selinux => {
    enabled => false
  }
}
```